projects/coordinator/src/test/java/org/batfish/coordinator/VersionCompatibilityFilterTest.java

```diff
@@ -0,0 +1,83 @@
+package org.batfish.coordinator;
```

Comment:
I don't understand "If it's present but wrong, we probably don't need to worry about it.
and same for API key". We only check whether the apiKey and version are provided in the header, without worrying about whether or not they are valid?